cadquery/selectors.py

```diff
@@ -38,24 +37,24 @@
     Keyword,
 )
 from functools import reduce
+from typing import List, Union, Sequence
 
 
 class Selector(object):
     """
-        Filters a list of objects
+    Filters a list of objects.
 
-        Filters must provide a single method that filters objects.
+    Filters must provide a single method that filters objects.
     """
 
     def filter(self, objectList):
         """
-            Filter the provided list
-            :param objectList: list to filter
-            :type objectList: list of FreeCAD primatives
-            :return: filtered list
-
-            The default implementation returns the original list unfiltered
+        Filter the provided list.
 
+        The default implementation returns the original list unfiltered.
+        :param objectList: list to filter
+        :type objectList: list of FreeCAD primatives
```

Comment:
Might as well change "FreeCAD" from the original here to OCCT. Also, "primatives" from the original should be "primitives".